pkg/controller/networkpolicy/networkpolicy_controller.go

```diff
@@ -507,6 +524,8 @@ func (n *NetworkPolicyController) processNetworkPolicy(np *networkingv1.NetworkP
 			Direction: networking.DirectionIn,
 			From:      *n.toAntreaPeer(ingressRule.From, np, networking.DirectionIn),
 			Services:  toAntreaServices(ingressRule.Ports),
+			Priority:  defaultRulePriority,
+			Action:    &defaultAction,
```

Comment:
why Action is optional if it's always set a default value?

Reply:
I think when ANP API types were merged this was kept as a pointer, so it stayed a pointer since, even we always assign it a value. We can circle back on this if necessary.